config/firestore.rules

```diff
@@ -0,0 +1,20 @@
+rules_version = '2';
+service cloud.firestore {
+  match /databases/{database}/documents {
+    match /datasets/covid19-som/metadata/{document=**} {
+    	allow write: if
+        request.auth != null &&
+      	request.auth.token.email_verified &&
```

Comment:
there's some weird indentation going on, I've had this before, I think the firebase rules editor switches to using tabs sometimes. This should fix it:
```suggestion
        request.auth.token.email_verified &&
```